### PR TITLE
adds name to readme

### PR DESCRIPTION
**Pull request by Ayowolet**

### CODE REVIEW

**Comment by datapythonista** [comment]
Thanks @Ayowolet, good job!

Do you mind in a separate pull request, make in the description `This is the list of the GitHub usernames of all the people interested in contributing to pandas:` the `GitHub` a link to the GitHub home page? Thank you!

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Mentoring new pandas contributors (team 2019)
 
-This is the list of the GitHub usernames of all the people interested in contributing to pandas:
+This is the list of the GitHub usernames of all the people interested in contributing to pandas: [Github Link](https://github.com/python-sprints/pandas-mentoring)
```
Comment (datapythonista) [inline README.md]: Sorry I wasn't clear, what I meant was more adding a link to the existing `GitHub` work, like:

```suggestion
This is the list of the [GitHub](https://github.com/) usernames of all the people interested in contributing to pandas:
```

Do you mind having both changes (adding your username, and adding this link) in separate pull requests? It's not trivial, let me know if you need help.

Comment (Ayowolet) [reply README.md]: okay, I misunderstood you. I will change it

Comment (datapythonista) [reply README.md]: No problem, my comment wasn't very clear, sorry about that. To change the link in this PR I think you should be able to do it directly from this page, with the option below "Suggested change". The tricky part is to have both changes in different PRs.

**Comment by datapythonista** [comment]
@Ayowolet not sure if you're having difficulties with this. A bit of information that may be useful:

In git, branches are used to have copies of the same repository to work on different things. `master` is the one that usually comes by default.

When you create a branch, you start from another branch. Because of that, what is common, is to never do any work in `master`. This way you'll always have a clean copy of the original repository where you can start new branches from.

Since you did changes in your master, you now can't create other branches to work in independent features, because the new branches will also have the changes you already made to master (and you don't want that).

So, what you probably want to do is to get a clean version of the repository first. This can be done with `git reset`, but it's probably easy just to clone the repository again.

Then you create branch with `git checkout -b add_username`, you make the changes, you commit, and you create a PR.

Then, you want to create a second branch, but you want it to be based on `master`, and not in `add_username`. So, you go back to master with `git checkout master`, and then you repeat the same you did before, but with a different branch name (e.g. `add_github_link`).

Hope this helps, let me know if you need extra help.

(Sorry if it was too much of a trouble trying it without the explanation, but I think it's usually easier to learn those things once you already faced all those problems you had).

**Pull request closed without merging**